internal/app/handler/shortener.go

```diff
@@ -0,0 +1,107 @@
+package handler
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"math/rand"
+	"net/http"
+
+	"github.com/FoGezz/go-url-shortener/cmd/shortener/config"
+	"github.com/FoGezz/go-url-shortener/internal/app/storage"
+	"github.com/go-chi/chi/v5"
+)
+
+var alphabet []rune = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
```

Comment:
Please don't use global variables

Reply:
The goal is not to allocate the alphabet every time I need it, Im gonna move it to cfg. But are there any better workarounds?